app/controllers/insights_cloud/api/machine_telemetries_controller.rb

```diff
@@ -76,9 +76,11 @@ def update_host_insights_status
       return unless request.path == '/redhat_access/r/insights/platform/ingress/v1/upload' ||
                     request.path.include?('/redhat_access/r/insights/uploads/')
 
-      data = @cloud_response.code.to_s.start_with?('2')
-      host_status = @host.get_status(InsightsClientReportStatus)
-      host_status.update(reported_at: Time.now.utc, status: host_status.to_status(data: data))
+      @cloud_response.code.to_s.start_with?('2')
```

Comment:
Do we need this anymore?

Reply:
Actually we do, we want to update only in case the code is in 200s. Will change the code accordingly